modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -170,6 +170,66 @@ public void testAddCustomObjectField() throws Exception {
 			() -> _addCustomObjectDefinitionWithEncryptedObjectField(
 				"AES", true, "",
 				ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT));
+		AssertUtils.assertFailure(
+			ObjectFieldBusinessTypeException.class,
+			"Salesforce storage type does not aggregation and attachment " +
```

Comment:
@kildarep the message seems incomplete. Wouldn't it be "Salesforce storage type does not **support** aggregation and attachment..."?

Reply:
@kildarep could you please update the message? Thanks!

Reply:
Updated messages. Thanks!